### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by Such6**
Updated the README.md file with development steps

ref: https://github.com/deepfence/yarahunter-docker-extension/issues/2

### CODE REVIEW

**Comment by ibreakthecloud** [comment]
@Such6 pls. reference the issue

File: README.md [hunk 1]
```diff
@@ -19,3 +19,8 @@ Enable Docker Extensions
 <img width="1377" alt="image" src="https://user-images.githubusercontent.com/18168330/194485464-ab8e402f-8e07-4a0c-86e5-3dae39623df8.png">
 
 <img width="1376" alt="image" src="https://user-images.githubusercontent.com/18168330/194485514-4991ccee-15a4-4df4-b407-c57a2a77d372.png">
+
+Setting up a new developer machine can be time consuming and tricky. The given steps for local development helps to simplify the process with easy-to-understand instructions as follows:
```
Comment (ibreakthecloud) [inline README.md]: can you put the exact steps from here (https://www.docker.com/blog/build-your-first-docker-extension/) under subsections like

## Development

### Enable hot reloading
```
cd ui
npm start
docker extension dev ui-source deepfenceio/yarahunter-extension http://localhost:3000
```

### Open Chrome Dev Tools
In order to open the Chrome Dev Tools for your extension when clicking on the extension tab, run:

`docker extension dev deepfenceio/yarahunter-extension`

Each subsequent click on the extension tab will also open Chrome Dev Tools. To stop this behavior, run:

`docker extension dev reset deepfenceio/yarahunter-extension`

### Change the backend service code
```
make build-extension
make update-extension
```

Comment (ibreakthecloud) [reply README.md]: and at then end put reference: https://www.docker.com/blog/build-your-first-docker-extension/

**Review by ibreakthecloud — CHANGES_REQUESTED** [review]

**Comment by Such6** [comment]
can you check now

File: README.md [hunk 2]
```diff
@@ -19,3 +19,24 @@ Enable Docker Extensions
...
+                                   
+2. enable hot reloading: to use local ui code without rebuilding
+   ## Insider Preview
+  <img width="1376" alt="image" src="https://user-images.githubusercontent.com/94823997/195160215-1d28d9af-cffe-4945-b33f-df41cf6582f2.png">
+
+ 3. Change the backend service code
+    make build-extension
+    make update-extension                            
+  
+ref: https://www.docker.com/blog/build-your-first-docker-extension/
```
Comment (ibreakthecloud) [inline README.md]: replace this with below


## Development

### Enable hot reloading
```
cd ui
npm start
docker extension dev ui-source deepfenceio/yarahunter-extension http://localhost:3000
```

### Open Chrome Dev Tools
In order to open the Chrome Dev Tools for your extension when clicking on the extension tab, run:

`docker extension dev deepfenceio/yarahunter-extension`

Each subsequent click on the extension tab will also open Chrome Dev Tools. To stop this behavior, run:

`docker extension dev reset deepfenceio/yarahunter-extension`

### Change the backend service code
```
make build-extension
make update-extension
```

File: README.md [hunk 3]
```diff
@@ -19,3 +19,42 @@ Enable Docker Extensions
...
+cd ui
+npm start
+docker extension dev ui-source deepfenceio/yarahunter-extension http://localhost:3000
+                                   
+
+## Insider Preview
+<img width="1376" alt="image" src="https://user-images.githubusercontent.com/94823997/195160215-1d28d9af-cffe-4945-b33f-df41cf6582f2.png">
+  
+  
+Open Chrome Dev Tools
```
Comment (ibreakthecloud) [inline README.md]:
```suggestion
### Open Chrome Dev Tools
```

File: README.md [hunk 4]
```diff
@@ -19,3 +19,44 @@ Enable Docker Extensions
...
+## DEVELOPMENT
+
+
+Enable hot reloading
+
+cd ui
+
+npm start
+
+docker extension dev ui-source deepfenceio/yarahunter-extension http://localhost:3000
```
Comment (ibreakthecloud) [inline README.md]:
```suggestion
### Enable hot reloading

cd ui
npm start
docker extension dev ui-source deepfenceio/yarahunter-extension http://localhost:3000
```

File: README.md [hunk 5]
```diff
@@ -19,3 +19,44 @@ Enable Docker Extensions
...
+                                   
+
+## Insider Preview
+<img width="1376" alt="image" src="https://user-images.githubusercontent.com/94823997/195160215-1d28d9af-cffe-4945-b33f-df41cf6582f2.png">
+
```
Comment (ibreakthecloud) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -19,3 +19,44 @@ Enable Docker Extensions
...
+## Insider Preview
+<img width="1376" alt="image" src="https://user-images.githubusercontent.com/94823997/195160215-1d28d9af-cffe-4945-b33f-df41cf6582f2.png">
+  
+  
+Open Chrome Dev Tools
+  
+In order to open the Chrome Dev Tools for your extension when clicking on the extension tab, run:
+
+docker extension dev deepfenceio/yarahunter-extension
```
Comment (ibreakthecloud) [inline README.md:41]:
```suggestion
`docker extension dev deepfenceio/yarahunter-extension`
```

File: README.md [hunk 7]
```diff
@@ -19,3 +19,44 @@ Enable Docker Extensions
...
+
+Each subsequent click on the extension tab will also open Chrome Dev Tools. To stop this behavior, run:
+
+docker extension dev reset deepfenceio/yarahunter-extension
```
Comment (ibreakthecloud) [inline README.md:45]:
```suggestion
`docker extension dev reset deepfenceio/yarahunter-extension`
```

File: README.md [hunk 8]
```diff
@@ -19,3 +19,44 @@ Enable Docker Extensions
...
+
+ ## Insider Preview
+
+<img width="1377" alt="image" src="https://user-images.githubusercontent.com/94823997/195160081-c939db84-3e17-458b-bc73-3c772a5e9534.png">
+
+
```
Comment (ibreakthecloud) [inline README.md]:
```suggestion
```

**Comment by Such6** [comment]
Do you want images to be removed, or it looks fine now

File: README.md [hunk 9]
```diff
@@ -19,3 +19,44 @@ Enable Docker Extensions
...
+Change the backend service code
```
Comment (ibreakthecloud) [inline README.md]:
```suggestion
### Change the backend service code
```

File: README.md [hunk 10]
```diff
@@ -19,3 +19,44 @@ Enable Docker Extensions
...
+ 
+ make build-extension
+    
+ make update-extension
```
Comment (ibreakthecloud) [inline README.md:53]:
```suggestion
 make build-extension
 make update-extension                            
```

**Review by ibreakthecloud — APPROVED** [review]
lgtm, thanks

**Pull request merged**